cvat-ui/src/components/header/settings-modal/shortcut-settings.tsx

```diff
@@ -53,7 +53,15 @@ function ShortcutsSettingsComponent(props: Props): JSX.Element {
                 if (currentSettings) {
                     try {
                         const parsedSettings = JSON.parse(currentSettings);
-                        delete parsedSettings.shortcuts;
+                        parsedSettings.shortcuts = {};
+                        for (const [key] of Object.entries(shortcuts.defaultState)) {
+                            if (key in shortcuts.defaultState) {
+                                parsedSettings.shortcuts.keyMap[key] = {
+                                    sequences: shortcuts.defaultState[key].sequences,
+                                };
+                            }
+                        }
```

Comment:
Code is duplicated with `onSaveSettings` callback.
Do we really need to store default shortcuts in the local storage? What is the motivation? 

Thus, if we decide to change default shortcut value in application, it won't be changed by default for anyone who uses CVAT with actually default shortcuts.

Reply:
Makes sense, removed